### ISSUE TITLE
[Bug]: Tricaster clearing tally status improperly

### ISSUE THREAD

**Issue by kolele** [josephdadams/TallyArbiter#623]
### What happened?

Take a look at the following video:

https://github.com/josephdadams/TallyArbiter/assets/19437233/9802801e-7adb-41c6-bfb6-78daf67157c1

Until fifth second of the video iam going through inputs 1-3 in preview bus. On seventh second of a video i press input 3 in PGM row on switcher. Issue: Tally status for preview is cleared.

Following video further, you can see me selecting inputs 3 than 2 and finally 1 in PGM row. On eleventh second i press input 1 in preview row. Issue: Tally status for program is cleared.

I suppose this is not intended behaviour?

### Version

3.0.7

### Distribution

Desktop

### OS

Windows

### What browsers are you seeing the problem on?

Chrome

**Comment by Jocke4f** [comment]
No, it is not how it should work. The code is currently assuming that the status from the Tricaster includes both program and preview information. Given that you have tested with a real Tricaster is that wrong, they are not always combined.

**Comment by kolele** [comment]
@Jocke4f thank you for looking into this!

Eagerly waiting for your merge request to complete so i can test.

**Comment by kolele** [comment]
Hello guys!

First test shows that tally data is being received properly from Tricaster! I will be definately testing more in upcoming week. Thank you for your efforts.

Two side questions:

1. Will there be any binaries which include this patch?
2. I cant seem to find any documentation on how reconnect system works. I see that i can edit configuration file where reconnect attempts can be modified. If i remember correctly in some previous version there was an option for reconnect interval?

Thanks once again!

**Comment by josephdadams** [comment]
This fix should be in the latest release.

**Comment by Jocke4f** [comment]
Your are correct that the fix for this issue is not yet in a release, the correction was submitted after 3.0.7 was released. There are a couple of fixes that hopefully will make a difference once 3.0.8 is released but I would like to sync up with Joseph if we should look into also fixing https://github.com/josephdadams/TallyArbiter/issues/640 for 3.0.8 or not.

    Den torsdag 1 februari 2024 kl. 08:25:46 CET, Slobodan Kosanovic ***@***.***> skrev:  
 
 


Hey @josephdadams thanks for taking your time to review this one!

Basically this fix came after 3.0.7 release and i have also tested 3.0.7 and can confirm that it does not contain it.

Source code does though.

Is there a chance we can get perhaps 3.0.8 with latest source code?

Thanks in advance for your time and efforts!

**Comment by josephdadams** [comment]
Yes we do need to fix that one

**Comment by kolele** [comment]
Great!

Excited for next binary release!

Thanks guys!
